content/docs/user-guide/scripting/script-canvas/editor-reference/text-replacement.md

```diff
@@ -1,126 +1,133 @@
 ---
-linkTitle: Text Replacement
+linkTitle: Node Text Replacement
 title: Script Canvas Node Text Replacement
 description: Learn how to customize the text on Script Canvas nodes in Open 3D Engine (O3DE).
 weight: 500
 ---
 
-# Script Canvas - Text Replacement User Documentation
+**Script Canvas** has a text replacement system that allows all available nodes to be further customized. Method names, categories, argument names, and tooltips can all be updated to improve their wording in a Script Canvas graph.
 
-**Script Canvas** has a text replacement system that allows all available nodes to be further customized. Method names, categories, argument names, and tooltips can all be updated to improve their look once in a Script Canvas graph.
+## How it works
 
-## Rules for updating text
-
-The `.names` files have certain fields that should not be modified, they are used by the system to create a key into a database of text fields. 
-
-**The following fields should NOT be changed**:
+When Script Canvas nodes are created, the developer can provide identifying and clarifying details such as the node's name, their category in the **Node Palette**, the names of their arguments and return values, and their tooltips. However, all of these details can be overridden using text replacement files.
 
-`base`
+Text replacement files use a JSON format and a file extension of `.names`. You can find all the text replacement files that were generated for the nodes that are included with O3DE in the directory `Gems/ScriptCanvas/Assets/TranslationAssets`.
 
-`context`
+You can generate or update `.names` files for new or existing nodes using **Script Canvas Editor**, as shown in the following sections.
 
-`variant`
+**Asset Processor** processes all the `.names` files for Script Canvas Editor to use in the Node Palette and when drawing the nodes in a graph.
 
-`typeid`
-
-Any changes to these fields will result in the Script Canvas Editor being unable to find the desired text to update and will result in the translation data being ignored.
+## Rules for updating text
 
-The data within the "details" field can be updated with the following fields:
+To customize a node in a `.names` file, you can update any of the following fields contained within any `details` field:
 
-`name`: The name of the node, argument, or return type.
+| Field | Description |
+| --- | --- |
+| `name` | The name of the node, argument, or return type. |
+| `category` | The category in the Node Palette to place the node in. Supports nested categories by using the "/" character. Example: "Gameplay/Missions". |
+| `tooltip` | Provides a tooltip to the element, this can be the node itself, or any of the node slots when the tooltip is added on a method or arguments. |
```

Comment:
Change to "argument" if it's singular, or something like "set of arguments" if it's plural. This is because it follows the article "a", to indicate singular. 
```suggestion
| `tooltip` | Provides a tooltip to the element. This can be the node itself, or any of the node slots when the tooltip is added on a method or argument. |
```